src/js/button.js

```diff
@@ -1,3 +1,8 @@
+import Component from './component';
+import * as VjsLib from './lib';
```

Comment:
Preferably, you wouldn't necessarily just import everything but rather the specific functions you need.